src/wui/load_or_save_game.cc

```diff
@@ -202,126 +191,28 @@ GameDetails* LoadOrSaveGame::game_details() {
 	return &game_details_;
 }
 
-const std::string LoadOrSaveGame::get_filename(uint32_t index) const {
-	return get_savegame(index).filename;
+void LoadOrSaveGame::select_item_and_scroll_to_it(std::set<uint32_t>& selections) {
+	const uint32_t selectme = *selections.begin();
```

Comment:
If you add

```
if (table_->empty()) {
    return;
}
```

on top, you can then have

`table_->select(std::min(selectme, table_->size() - 1));`

Reply:
yeah, nice, thanks :)